test/library/library.go

```diff
@@ -32,3 +47,105 @@ func GenerateNameForTest(t *testing.T, prefix string) string {
 	// ensure no `-` at beginning or end
 	return strings.Trim(string(name), "-")
 }
+
+// WaitForAPIServerToBeOnTheSameRevision waits until all API Servers are running at the same revision.
+// The API Servers must stay on the same revision for at least waitForAPIRevisionSuccessThreshold * waitForAPIRevisionSuccessInterval.
+// Mainly because of the difference between the propagation time of triggering a new release and the actual roll-out.
+//
+// Observations:
+//  rolling out a new version is not instant you need to account for a propagation time (~1/2 minutes)
+//  rolling out a new version can take ~10 minutes
+func WaitForAPIServerToBeOnTheSameRevision(t *testing.T, podClient corev1client.PodInterface) {
```

Comment:
WaitForAPIServerTo*Stabilize*OnTheSameRevision ? This is not a no-op if there is no rollout started yet. So the current name is misleading. It waits at least 3 min, giving the operator time to deploy.

Reply:
done